compose/ui/ui/src/uikitMain/parse_crowdin.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#
+# Copyright 2024 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Directory to go through
```

Comment:
Let's move this file to the top `/scripts` and rename to `convertCrowdinToStringsXml`

Reply:
Done